Library/Formula/sonar.rb

```diff
@@ -46,4 +44,8 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    assert_match /SonarQube/, pipe_output("#{bin}/sonar status")
```

Comment:
I had some errors when trying to get the sonar4 test working in
homebrew-versions this is a direct copy of the test implemented there.

I will take a look again, trying to keep the scripts the same as much as
possible.

On 1 Mar 2015, at 21:47, Mike McQuaid <notifications@github.com> wrote:

In Library/Formula/sonar.rb
<https://github.com/Homebrew/homebrew/pull/37237#discussion_r25571910>:

> @@ -46,4 +44,8 @@ def plist; <<-EOS.undent
>      </plist>
>      EOS
>    end
> +
> +  test do
> +    assert_match /SonarQube/, pipe_output("#{bin}/sonar status")

Use shell_output here.

—
Reply to this email directly or view it on GitHub
<https://github.com/Homebrew/homebrew/pull/37237/files#r25571910>.